src/components/Transitions.js

```diff
@@ -0,0 +1,59 @@
+import generate from "@babel/generator"; // babel-generator
+// import type { Transition } from "./types";
+
+/* type Parent = {
+    isProgram: () => boolean,
+    parentPath: Parent,
+    node: {
+        type: string,
+    },
+}; */
+
+export default class Transitions {
+    _transitions = []; //: Array<Transition>
+
+    _getProgramParent = (path) => { // Parent
+        let parent = path;
+        do {
+            if (parent.isProgram()) return parent;
+        } while ((parent = parent.parentPath));
+    };
+
+    getValue = () => {
+        return this._transitions;
+    };
+
+    addExitTransition = (code) => { // string
+        this._transitions.push({
+            code,
+            pluginAlias: "output",
+            visitorType: "exit",
+            size: new Blob([code], { type: "text/plain" }).size,
+        });
+    };
+
+    wrapPluginVisitorMethod = (
+        pluginAlias, // : any
+        visitorType, // : any
+        callback // : any
+    ) => {
+        return (...args) => { // : any
+            // $FlowFixMe
```

Comment:
Is this using Flow?

Reply:
Nope, removed all the flow, left the comments just for reference, but will remove